zebra/label_manager.c

```diff
@@ -148,46 +173,81 @@ static int reply_error(int cmd, struct zserv *zserv, vrf_id_t vrf_id)
  * @return 0 on success, -1 otherwise
  */
 int zread_relay_label_manager_request(int cmd, struct zserv *zserv,
-				      vrf_id_t vrf_id)
+							struct stream *msg, vrf_id_t vrf_id)
```

Comment:
Can you please align this line like this:
```
int zread_relay_label_manager_request(int cmd, struct zserv *zserv,
                                      struct stream *msg, vrf_id_t vrf_id)
```
That is, the follow-on parameter line aligned with the first character of the first parameter in the first line, indented with tabs and aligned with spaces like so:
```
example_foo(a, b,
[tab][tab]  c)
```
We'll patch our style checker to catch these kinds of stylistic issues in the future. Sorry for the trouble.